privval/signer_endpoint.go

```diff
@@ -94,14 +96,15 @@ func (se *signerEndpoint) ReadMessage() (msg SignerMessage, err error) {
 		return
 	}
 
-	const maxRemoteSignerMsgSize = 1024 * 10
```

Comment:
please leave the const https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants